src/data/symbol/symbol.tsv

```diff
@@ -509,6 +515,63 @@ POS	CHAR	Reading (space separated)	description	additional description	category	m
 記号	🀩	まーじゃん まーじゃんぱい ふゆ	麻雀牌	冬	SYMBOL
 記号	🀪	まーじゃん まーじゃんぱい ぱいたー じょーかー	麻雀牌	百搭	SYMBOL
 記号	🀫	まーじゃん まーじゃんぱい まーじゃんうら	麻雀牌	裏面	SYMBOL
+記号	🂠	とらんぷ とらんぷうら	トランプ	裏面	SYMBOL
+記号	🂡	とらんぷ すぺーどA すぺーど1 すぺーどえーす	トランプ	スペードA	SYMBOL
+記号	🂢	とらんぷ すぺーど2	トランプ	スペード2	SYMBOL
+記号	🂣	とらんぷ すぺーど3	トランプ	スペード3	SYMBOL
+記号	🂤	とらんぷ すぺーど4	トランプ	スペード4	SYMBOL
+記号	🂥	とらんぷ すぺーど5	トランプ	スペード5	SYMBOL
+記号	🂦	とらんぷ すぺーど6	トランプ	スペード6	SYMBOL
+記号	🂧	とらんぷ すぺーど7	トランプ	スペード7	SYMBOL
+記号	🂨	とらんぷ すぺーど8	トランプ	スペード8	SYMBOL
+記号	🂩	とらんぷ すぺーど9	トランプ	スペード9	SYMBOL
+記号	🂪	とらんぷ すぺーど10	トランプ	スペード10	SYMBOL
+記号	🂫	とらんぷ すぺーどJ すぺーど11 すぺーどじゃっく	トランプ	スペードJ	SYMBOL
+記号	🂬	すぺーどC すぺーどないと かばろ すぺーどかばろ 	トランプ	スペードC/カバロ	SYMBOL
```

Comment:
> Would you share us some references for "カバロ"?

https://gamefarm.jp/rule/cego.html

> 　4つのスートは、普通のトランプと同様に、スペード、クラブ、ダイヤモンド、ハートです。（占い用のカードを使うときには、スペード=剣、クラブ=棍棒、ダイアモンド=貨幣、ハート=聖杯として使ってください。）
> \*snip\*
> （占い用のカードでは、カバロのことをナイトと呼んでいるかもしれません。）

https://boardgameblog.net/2023/01/09/gametarot/

> ナイト（カバロ）の絵札がジョーカー表記になってしまっている
>
> 4つのスートすべてのナイト（カバロ表記のタロットも多い）が、ジョーカー扱いになっているのです。これではタロットのゲームで使えません。タロットは絵札がJ・N（C）・Q・Kの4枚あるはずなのです。
> \*snip\*
> こちらはナイトではなくカバロ表記でした。

http://www.circle9.work/cards/intro/tarots.html

> ゲーム用 | Jジャック(Jack) | Cカバロ(Cavallo) | Qクイーン(Queen) | Kキング(King)
> -- | -- | -- | -- | --
> Pペイジ(Page) | Ktナイト(Knight) | Qクイーン(Queen) | Kキング(King)
> Vバレ(Valet) | Cカバリエ(Cavalier) | Dダム(Dam) | Rルワ(Roi)

https://cardgame.wiki.fc2.com/wiki/%E3%82%BF%E3%83%AD%E3%83%83%E3%83%88%E6%A6%82%E8%AB%96

> なお、占い用のタロットカードはカバロがナイト(knight)になり、ジャックがペイジ(page)になります。

Therefore, these "C" suite should correspond to "カバロ", not "ナイト" in Japanese. If it corresponded to "ナイト", the alphabet would have to be "N" or "Nt".

> For the consistency with other description, only "スペードC" looks good?

Since it can't be converted from "とらんぷ" now, it's fine. It's dedicated to those who know trick-taking game.

Reply:
Thank you for the explanation. These references will be helpful for the code readers in future.